src/shared/local/localShadowRepo.ts

```diff
@@ -161,6 +153,11 @@ export class ShadowRepo {
           filter: fileFilter(this.packageDirs),
         });
 
+        // isomorphic-git stores things in unix-style tree.  Convert to windows-style if necessary
+        if (IS_WINDOWS) {
```

Comment:
this needed to be above moveDetection so that all of that code could have deterministic paths based on the OS (because SDR Resolver is os-dependent)